### PR TITLE
Spec changes for agent and loadbalancer

### PR DESCRIPTION
**Pull request by ramaraosrikakulapu**
Below are the summary of changes - 

_Agent_: 
- Adding new header 'CF_INSTANCE_INDEX' for k8s environment

_Loadbalancer_: 
- Updated access log path for load balancer
- Updated logic for replacing nginx 'server block' configuration
- Updated nginx config to allow web sockets, allow underscores and custom logs

### CODE REVIEW

**Comment by ramaraosrikakulapu** [comment]
@ayasuda-ge .. Can you please review the code and **squash merge** the changes, if everything looks good

File: scripts/agt/v1.linux64_conf.txt [hunk 1]
```diff
@@ -61,4 +61,9 @@ ls -la ~/.ec/agt/bin
 ln -s ~/.ec/agt/bin/${AGENT_REV_LOCAL} ~/.ec/agent
 
 export PATH=$PATH:$HOME/.ec
+
```
Comment (ayasuda-ge) [inline scripts/agt/v1.linux64_conf.txt:64]: line 65-68 should stay in ec-nginx.sh

**Comment by ramaraosrikakulapu** [comment]
@ayasuda-ge .. Can you please review the code and squash merge the changes, if all looks good. Please let me know for any questions.

File: scripts/agt/v1.linux64_conf.txt (hunk 1 above)
Comment (ayasuda-ge) [reply scripts/agt/v1.linux64_conf.txt:64]: can we move this logic to https://github.com/EC-Release/sdk/blob/bc64668b8014b89713199580b289d9da597d7ef0/scripts/agt/conf/agent_v1.sh @ramaraosrikakulapu

Comment (ramaraosrikakulapu) [reply scripts/agt/v1.linux64_conf.txt:64]: Implemented the requested change @ayasuda-ge

**Review by ayasuda-ge — APPROVED** [review]

**Pull request merged**